x-pack/plugins/stack_connectors/server/connector_types/security/tines/api_schema.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import { TinesStoryObjectSchema } from '../../../../common/connector_types/security/tines/schema';
+
+// Tines response base schema
+export const TinesBaseApiResponseSchema = schema.object(
+  {
+    meta: schema.object(
+      {
+        next_page: schema.nullable(schema.string()),
+      },
+      { unknowns: 'ignore' }
```

Comment:
It doesn't appear we use this option for other connector schemas.  I think I'd advise against it.  

With it set this way, a customer may pass some bogus property in their params/secrets/etc, and today we'd ignore it.  If we later add more properties to our schema, it's possible they were already using that property name, and the connector could break when it runs.  A type of migration failure.  We had a similar situation a while back, required a migration to fix it.

Reply:
hmmm ... I just realized these are schemas for the Tine's HTTP requests/responses, so ignore that comment ^^^ :-)

Reply:
yes exactly, these are API response schemas, they are a bit wider to prevent throwing errors if there's more information than needed, everything should keep working.